ui/src/websites/artconomy/ArtconomyLogin.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { LoginDialogProps } from '../interfaces/website.interface';
+import { Form, Input, Button, message } from 'antd';
+import LoginService from '../../services/login.service';
+import Axios, {AxiosError} from 'axios';
+
+interface State {
+    email: string;
+    password: string;
+    token: string;
+    show2fa: boolean,
+    sending: boolean;
+}
+
+export default class ArtconomyLogin extends React.Component<LoginDialogProps, State> {
+    state: State = {
+        email: '',
+        password: '',
+        token: '',
+        show2fa: false,
+        sending: false,
+    };
+
+    constructor(props: LoginDialogProps) {
+        super(props);
+        this.state = {
+            ...this.state,
+            ...(props.data as State)
+        };
+    }
+
+    async submit() {
+        this.setState({ sending: true });
+        const auth = await Axios.post<{ id: number, username: string, csrftoken: string, token: string[] }>(
+            `${window.electron.environment.ARTCONOMY_URL}/api/profiles/v1/login/`,
+            {...this.state},
+            { responseType: 'json' },
```

Comment:
So I noticed that the actual website uses the same API you are calling here.
Wouldn't it be better to just allow the user to login through the UI like most of the other websites.

I haven't logged in for real, but I imagine you store those auth tokens somewhere like localStorage or something which can be grabbed from the nodeJS side similar to how FurryNetwork is implemented.